client/src/pages/organizations/Laydown.js

```diff
@@ -35,7 +36,7 @@ class BaseOrganizationLaydown extends Component {
 		const positionsNeedingAttention = org.positions.filter(position => !position.person )
 		const supportedPositions = org.positions.filter(position => positionsNeedingAttention.indexOf(position) === -1)
 
-		return <div id="laydown" data-jumptarget>
```

Comment:
Also remove the `data-jumptarget` attribute from `client/src/pages/insights/Show.js`.